.cmake-format.yaml

```diff
@@ -0,0 +1,5 @@
+dangle_parens: true
+first_comment_is_literal: true
+keyword_case: 'upper'
+line_width: 120
+tab_size: 4
```

Comment:
How about customize this configuration file to minimize changes to existing cmake files?